docs/rules/no-unsupported-features/node-builtins.md

```diff
@@ -49,273 +49,16 @@ The `"allowExperimental"` option accepts a boolean value (the default value is `
-- `"fs.copyFileSync"`
-- `"fs.mkdtemp"`
-- `"fs.mkdtempSync"`
-- `"fs.realpath.native"`
-- `"fs.realpathSync.native"`
-- `"fs.promises"`
-- `"fs.writev"`
-- `"fs.writevSync"`
-
-**`http2` module:**
-
-- `"http2"`
-
-**`inspector` module:**
-
-- `"inspector"`
-
-**`module` module:**
-
-- `"module.Module.builtinModules"`
-- `"module.Module.createRequireFromPath"`
-- `"module.Module.createRequire"`
-- `"module.Module.syncBuiltinESMExports"`
-- `"module.builtinModules"`
-- `"module.createRequireFromPath"`
-- `"module.createRequire"`
-- `"module.syncBuiltinESMExports"`
-
-**`os` module:**
-
-- `"os.constants"`
-- `"os.constants.priority"`
-- `"os.getPriority"`
-- `"os.homedir"`
-- `"os.setPriority"`
-- `"os.userInfo"`
-
-**`path` module:**
-
-- `"path.toNamespacedPath"`
-
-**`perf_hooks` module:**
-
-- `"perf_hooks"`
-- `"perf_hooks.monitorEventLoopDelay"`
-
-**`process` module:**
-
-- `"process.allowedNodeEnvironmentFlags"`
-- `"process.argv0"`
-- `"process.channel"`
-- `"process.cpuUsage"`
-- `"process.emitWarning"`
-- `"process.getegid"`
-- `"process.geteuid"`
-- `"process.hasUncaughtExceptionCaptureCallback"`
-- `"process.hrtime.bigint"`
-- `"process.ppid"`
-- `"process.release"`
-- `"process.report"`
-- `"process.resourceUsage"`
-- `"process.setegid"`
-- `"process.seteuid"`
-- `"process.setUncaughtExceptionCaptureCallback"`
-- `"process.stdout.getColorDepth"`
-- `"process.stdout.hasColor"`
-- `"process.stderr.getColorDepth"`
-- `"process.stderr.hasColor"`
-
-**`stream` module:**
-
-- `"stream.Readable.from"`
-- `"stream.finished"`
-- `"stream.pipeline"`
-
-**`trace_events` module:**
-
-- `"trace_events"`
-
-**`url` module:**
-
-- `"url.URL"`
-- `"url.URLSearchParams"`
-- `"url.domainToASCII"`
-- `"url.domainToUnicode"`
-
-**`util` module:**
-
-- `"util.callbackify"`
-- `"util.formatWithOptions"`
-- `"util.getSystemErrorName"`
-- `"util.inspect.custom"`
-- `"util.inspect.defaultOptions"`
-- `"util.inspect.replDefaults"`
-- `"util.isDeepStrictEqual"`
-- `"util.promisify"`
-- `"util.TextDecoder"`
-- `"util.TextEncoder"`
-- `"util.types"`
-- `"util.types.isBoxedPrimitive"`
-
-**`v8` module:**
-
-- `"v8"`
-- `"v8.DefaultDeserializer"`
-- `"v8.DefaultSerializer"`
-- `"v8.Deserializer"`
-- `"v8.Serializer"`
-- `"v8.cachedDataVersionTag"`
-- `"v8.deserialize"`
-- `"v8.getHeapCodeStatistics"`
-- `"v8.getHeapSnapshot"`
-- `"v8.getHeapSpaceStatistics"`
-- `"v8.serialize"`
-- `"v8.writeHeapSnapshot"`
-
-**`vm` module:**
-
-- `"vm.Module"`
-- `"vm.compileFunction"`
-
-**`worker_threads` module:**
-
-- `"worker_threads"`
-
-</details>
+The `"ignores"` option accepts an array of strings.
+
+> [!TIP]
+> You will see the following error messages:
+>
+> - `The '{{name}}' is not an experimental feature until Node.js {{version}}.`
+> - `The '{{name}}' is still an experimental feature and is not supported until Node.js {{version}}.`gi
+> - `The '{{name}}' is still an experimental feature`
+>
+> The "name" property is what you can use in your `ignores` array
```

Comment:
maybe adding a link to the modules: https://github.com/eslint-community/eslint-plugin-n/tree/HEAD/lib/unsupported-features/node-builtins-modules?

Reply:
I like that idea.
```suggestion
> The "name" property is what you can use in your `ignores` array
> You can find the full tree view of the list of the modules over in [lib/unsupported-features/node-builtins-modules](https://github.com/eslint-community/eslint-plugin-n/tree/HEAD/lib/unsupported-features/node-builtins-modules). and globals over in [lib/unsupported-features/node-globals.js](https://github.com/eslint-community/eslint-plugin-n/blob/HEAD/lib/unsupported-features/node-globals.js)
```